tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkBuffer.java

```diff
@@ -67,6 +67,12 @@ public ChunkBuffer(MeasurementSchema schema) {
     this.pageBuffer = new PublicBAOS();
   }
 
+  public ChunkBuffer(MeasurementSchema schema, PublicBAOS pageBuffer) {
```

Comment:
change the type of the second parameter to ListPublicBAOS

Reply:
Good comment.

Should `ChunkBuffer` perceive `ListPublicBAOS`?  `ChunkBuffer` and  `PublicBAOS` are designed for `TsFile`,  while `ListPublicBAOS` is introduced to optimize IoTDB engine. 

I think we could keep the type of the second parameter as `PublicBAOS`.

Reply:
Good comment.

Should `ChunkBuffer` perceive `ListPublicBAOS`?  `ChunkBuffer` and  `PublicBAOS` are designed for `TsFile`,  while `ListPublicBAOS` is introduced to optimize IoTDB engine. 

I think we could keep the type of the second parameter as `PublicBAOS`.